lib/modules/commentNavigator.js

```diff
@@ -89,7 +88,7 @@ modules['commentNavigator'] = {
 						case 'images':
 							thisEle.setAttribute('title', 'Navigate through comments with images');
 							break;
-						case 'images':
+						case 'videos':
```

Comment:
So I just tried `makelinks.sh` again, a little more carefully (closing Chrome first :P), and was able to reproduce it.

Then I managed to find [this issue report](https://youtrack.jetbrains.com/issue/IDEA-78480), and followed its suggestion to turn off Settings > Appearance & Behavior > System Settings > Use "safe write" - and voila, it worked!